extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -478,18 +484,34 @@ private FullResponseHolder submitRequest(
         throw new TaskNotRunnableException(String.format("Aborting request because task [%s] is not runnable", id));
       }
 
+      String host = location.getHost();
```

Comment:
nit: i think you could make these final and set them later, since they aren't used until after their reassignment at line 497